skrifa/src/outline/hint.rs

```diff
@@ -28,6 +198,7 @@ use crate::alloc::{boxed::Box, vec::Vec};
 /// FreeType 2.7.
 ///
 /// The default value of this type is equivalent to `FT_LOAD_TARGET_NORMAL`.
+#[doc(hidden)]
```

Comment:
Should this be `#[deprecated]`?

Reply:
Yes, but we require transitional states for rolling updates in Chrome and I specifically avoided the deprecated attribute to avoid raising warnings at compile time.

Reply:
Thanks for the explanation. I wonder if whatever uses this in chrome should use [`allow(deprecated)`](https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#deprecated) instead. I think that should be entirely equivalent for that use case.

I suppose that could cause problems if a non-skrifa library were updated?

Reply:
Yeah, it seems like that could affect other things in undesirable ways. My plan was to mark these deprecated for the next cycle and then remove them which adds a bit of latency but should be fine.